canvas2D/src/Engine/babylon.text2d.ts

```diff
@@ -132,6 +132,14 @@
         }
     }
 
+    export enum TextAlign {

```

Comment:
We are not allowed to use enum or const enum, you have to realy on a static readonly number. right @deltakosh (because I have a doubt concerning const enum) ?